apps/meteor/ee/app/livechat-enterprise/server/api/tags.ts

```diff
@@ -3,7 +3,7 @@ import { findTags, findTagById } from './lib/tags';
 
 API.v1.addRoute(
 	'livechat/tags',
-	{ authRequired: true },
+	{ authRequired: true, permissionsRequired: ['view-l-room'] },
```

Comment:
Missing permission: `manage-livechat-tags`

Reply:
Also, since we have a permission check here, so can we remove it from `findTags` to avoid re-checking?